Makefile

```diff
@@ -1,4 +1,4 @@
-CFLAGS += -Wall
+CFLAGS += -Wall -Werror -Wextra
```

Comment:
*Never* use `-Werror` as a default setting - this *will* break builds in the future.

Reply:
Using it in CI is fine, but it must be injected through the CI file, not the Makefile.

New compiler versions will introduce new warnings - this should be caught by the CI, but it must not break builds otherwise.

Reply:
Oh, indeed that's a fair distinction, I'll fix this tonight; thanks!

Reply:
resolved in ae493973fc2ba551f28a02d95403c689ee807b69